From 5dc520347c1a557fea4ce4633e704091bfe90d66 Mon Sep 17 00:00:00 2001
From: Jihoon Bang <jbang@nvidia.com>
Date: Wed, 15 May 2013 09:15:48 -0700
Subject: [PATCH 10/10] power: add camera power hints

Add still preview, video preview and video record
hint to camera hint.

Bug 1232396

Change-Id: I75060e28f6ef82724649f17f46b125bf29a650b7
Reviewed-on: http://git-master/r/228931
Reviewed-by: Simone Willett <swillett@nvidia.com>
Tested-by: Simone Willett <swillett@nvidia.com>
---
 include/hardware/power.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index ae96aeb..0cfcd24 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -71,7 +71,9 @@ typedef enum {
  */
 
 typedef enum {
-    CAMERA_HINT_POWER,
+    CAMERA_HINT_STILL_PREVIEW_POWER,
+    CAMERA_HINT_VIDEO_PREVIEW_POWER,
+    CAMERA_HINT_VIDEO_RECORD_POWER,
     CAMERA_HINT_PERF,
     CAMERA_HINT_FPS,
     CAMERA_HINT_RESET,
-- 
1.8.1.5

